### PR TITLE
add stat folder

### PR DESCRIPTION
**Pull request by Lindros9203**

### CODE REVIEW

File: stat/python_script_lm_predictors.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+# compile the keras model
+
+opt = keras.optimizers.Adam(learning_rate=0.001)
+
+model.compile(loss='mae', optimizer=opt, metrics=['mae'])
+
+# fit the keras model on the dataset
+
+m=model.fit(train_input,train_output,epochs=30000,verbose=1,steps_per_epoch=None)
```
Comment (xkadjan) [inline stat/python_script_lm_predictors.py:38]: Z mých zkušeností je takto "velký" parametr learning rate u kompilátoru Adam v kombinaci s daným počtem epoch teměř jistota přeučení. 
Pokud to funguje - ok. Jen bych radeji videl nejake grafy prubehu uceni (TensorBoard)

File: stat/readme.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Zajímavým úkazem je, že pokud se zvolí šikovnné "seed number", tak lineární model dává přesnější odhady než
+neuronová síť. Pozoroval jsem to asi dvakrát, kdy mi to vyšlo úplně náhodou. Otázkou je, zda tedy neudělat sérii
+běhů (třeba 100), přičemž při každém by se náhodně vybrala testovací a trénovací data z celého datasetu a pak se 
+statisticky porovnávaly odchylky. To by podle mého byla asi metodicky nejzajímavější varianta.
```
Comment (xkadjan) [inline stat/readme.txt:4]: Určitě zajímavý nápad.
Nemám zkušenosti s lineárním modelem, ale obecně ta neuronová síť se mi jeví jako obrovský kanon. S nasim malym datasetem to upravdu může být spíš náhoda...

**Review by xkadjan — APPROVED** [review]
